src/test/java/org/kohsuke/github/GHRepositoryTest.java

```diff
@@ -823,6 +823,33 @@ public void ghRepositorySearchBuilderForkDefaultResetForksSearchTerms() {
         assertThat(ghRepositorySearchBuilder.terms.stream().filter(item -> item.contains("fork:")).count(), is(0L));
     }
 
+    /**
+     * Test createFork method with valid parameters.
+     *
+     * @throws IOException
+     *             Signals that an I/O exception has occurred.
+     */
+    @Test
+    public void testCreateForkWithValidParameters() throws IOException {
```

Comment:
Take a look at the test for `forkTo()`: https://github.com/hub4j/github-api/blob/58dcca18195dc7fc8614e1c2c39fac37c3bf2ae3/src/test/java/org/kohsuke/github/AppTest.java#L693-L697

That might help.